Docs Java and other small fixes and clarifications (#12746)
metabase/metabase: docs/administration-guide/01-managing-databases.md

Now you’ll see a list of your databases. To connect another database to Metabase, click **Add database**. Metabase currently supports the following types of databases:

- Amazon Redshift
- Druid
- Google Analytics
- [Google BigQuery](databases/bigquery.md)
- H2
- [MongoDB (version 3.4 or higher)](databases/mongodb.md)
- [MySQL (version 5.5.3 or higher, as well as MariaDB)](databases/mysql.md)
- [Oracle](databases/oracle.md)

> - Postgres

Reviewer: ```suggestion
- PostgreSQL
```

#### Scanning for field values

> When Metabase first connects to your database, it takes a look at the metadata of the fields in your tables and automatically assigns them a field type. Metabase also takes a sample of each table to look for URLs, JSON, encoded strings, etc. If a field is classified wrong, you can always manually edit it from the **Data Model** tab in the Admin Panel.

Reviewer: ```suggestion
When Metabase first connects to your database, it takes a look at the metadata of the columns in your tables and automatically assigns them a type. Metabase also takes a sample of each table to look for URLs, JSON, encoded strings, etc. You can manually edit table and column metadata in Metabase at any time from the **Data Model** tab in the **Admin Panel**.
```
